### PR TITLE
Lovefield.d.ts : the timing option in RawForeignKeySpec type is wrong

### PR DESCRIPTION
**Pull request by Kornik34**
Actually, in RawForeignKeySpec, the options "action" and "timing" are both of type ConstraintAction. The "timing" option should be of type "ConstraintTiming" like this:

type RawForeignKeySpec = {
local: string
ref: string
action: lf.ConstraintAction
timing: lf.ConstraintTiming
}

### CODE REVIEW

File: request-promise/request-promise.d.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ declare module 'request-promise' {
     module RequestPromiseAPI {
         export interface Options extends request.Options {
             simple?: boolean;
-            transform?: (body: any, response: http.IncomingMessage) => number;
+            transform?: (body: any, response: http.IncomingMessage) => any;
```
Comment (joeskeen) [inline request-promise/request-promise.d.ts]: This is more correct than `number`, thanks for catching this.

File: angularjs-toaster/angularjs-toaster.d.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ declare module ngtoaster {
     error(params: IPopParams): void
     error(title?:string, body?:string, timeout?:number, bodyOutputType?:string, clickHandler?:EventListener,
         toasterId?:number): void
-    into(params: IPopParams): void
+    info(params: IPopParams): void
```
Comment (btesser-r7) [inline angularjs-toaster/angularjs-toaster.d.ts]: LGTM

**Comment by btesser-r7** [comment]
:+1: LGTM

**Comment by vvakame** [comment]
@Kornik34 please resolve conflict.

**Comment by Kornik34** [comment]
My PR was about Lovefield.d.ts, not angularjs-toaster.d.ts...
LGTM

**Pull request closed without merging**